docs/roadmap.md

```diff
@@ -1,56 +1,47 @@
 ## KICS Roadmap
 
-| Status | Milestone | ETA | Target Release |
-| :--- | :--- | :--- | :--- |
-| Not-Started | **Milestone 8** | Jun 2021 | |
-|  | Google Deployment Manager support |  | TBA |
-| Not-Started | **Milestone 7** | May 2021 | |
-|  | Schemas support OpenAPI 3.0|  | TBA |
-|  | Schemas support OpenAPI 2.0 (Swagger) |  | TBA |
-|  | Schemas support gRPC |  | TBA |
-|  | Azure Resource Manager support |  | TBA |
+These are our upcoming features:
+- Google Deployment Manager support
```

Comment:
Please, move `Google Deployment Manager support` to the bottom of this list (since it is the lower priority)